ROADMAP.md

```diff
@@ -1,4 +1,4 @@
-## Roadmap
+## Roadmap 🗺
```

Comment:
I think the 🌐 is a better symbol for this.

Reply:
Maybe you are right!